README.md

```diff
@@ -98,6 +106,28 @@ Create a new lion instance with the name `simba`
 
 ```ruby
 # code here
+class Animal
+  attr_accessor :name,
+  @@pack = "Lion"
+
+  def initialize name
+    @name = name
+    @king = true
+  end
+  def greet
+    Puts "Hi #{name}"
+  end
+end
+animal = new Animal "Pumba"
+
+class Lion < Animal
+  def greet
+    puts @@pack
+  end
+
+end
+
+lion = new Lion "simba"
 ```
```

Comment:
close, I think you have some of the functionality (i.e. `pack`, `king`) in the `Animal` class, when it should be in `Lion`